README.md

```diff
@@ -374,15 +374,18 @@ TODO
 
 ### `scaling`
 
-TODO
+A simple OpenShift test verifying that the OpenShift deployment with Quarkus application scales up and down.
```

Comment:
```suggestion
An OpenShift test verifying that the OpenShift deployment with Quarkus application scales up and down.
```

Reply:
I'd even suggest just "Verifies that an OpenShift deployment with Quarkus application scales up and down."

Reply:
And perhaps add that this could be expanded in the future to collect some performance metrics -- but that's hard to do in a meaningful way, because the OpenShift overhead is much bigger than Quarkus startup time.